<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_22">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>160</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Current Cordinates</string>
    </property>
    <widget class="QLCDNumber" name="lcdLat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdLon">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_23">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1151</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string>Mapa</string>
    </property>
    <widget class="QQuickWidget" name="quickWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1131</width>
       <height>711</height>
      </rect>
     </property>
     <property name="resizeMode">
      <enum>QQuickWidget::SizeRootObjectToView</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_24">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>MQTT</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>64</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leHost">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>164</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>broker.hivemq.com</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="topic">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>45</y>
       <width>164</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TrackGPS/gpsData</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MQTT Broker</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Stopped</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>47</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Topic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="runMQTTButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_25">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>570</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Perimeter Alarm</string>
    </property>
    <widget class="Led" name="led" native="true">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked" stdset="0">
      <bool>false</bool>
     </property>
     <property name="color" stdset="0">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
    <widget class="QPushButton" name="circleButton_ONOFF">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>On/off</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="circleSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="circleLcd">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_26">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>290</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Speed</string>
    </property>
    <widget class="QLCDNumber" name="lcdSpeed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>km/h</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_27">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>290</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Battery Level</string>
    </property>
    <widget class="QLCDNumber" name="lcdBatPerc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>114</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdBatVolt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>54</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>34</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltage</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_28">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>650</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Speed Alarm</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>km/h</string>
     </property>
    </widget>
    <widget class="Led" name="ledSpeed" native="true">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked" stdset="0">
      <bool>false</bool>
     </property>
     <property name="color" stdset="0">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
    <widget class="QPushButton" name="speedButton_ONOFF">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>On/off</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="speedLcd">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="speedSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_29">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>450</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Route</string>
    </property>
    <widget class="QPushButton" name="visibleTrackButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>On/off</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Show route</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset route</string>
     </property>
    </widget>
    <widget class="QPushButton" name="visibleTrackButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_25</zorder>
   <zorder>groupBox_22</zorder>
   <zorder>groupBox_24</zorder>
   <zorder>groupBox_23</zorder>
   <zorder>groupBox_26</zorder>
   <zorder>groupBox_27</zorder>
   <zorder>groupBox_28</zorder>
   <zorder>groupBox_29</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>analogwidgets/led.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>speedSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>speedLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1627</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>1668</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>circleSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>circleLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1675</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>1799</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
